src/projects/gt4o_us/setting/setting.ad

```diff
@@ -0,0 +1,419 @@
+	TURBINEKIT_list[5] = ["TURBINE_KIT_STAGE5"];
+	
+	static PORTPOLISH_list = Array(2);
+	PORTPOLISH_list[1] = ["PORTPOLISH"];
+	
+	static ENGINEBALANCE_list = Array(2);
+	ENGINEBALANCE_list[1] = ["ENGINEBALANCE"];
+	
+	static DISPLACEMENT_list = Array(2);
+	DISPLACEMENT_list[1] = ["DISPLACEMENT"];
+	
+	static COMPUTER_list = Array(2);
+	COMPUTER_list[1] = ["COMPUTER"];
+	
+	static INTERCOOLER_list = Array(3);
+	INTERCOOLER_list[1] = ["INTERCOOLER_SPORTS"];
+	INTERCOOLER_list[2] = ["INTERCOOLER_RACING_BIG"];
+	
+	static MUFFLER_list = Array(4);
+	MUFFLER_list[1] = ["SPORTS_MUFFLER"];
+	MUFFLER_list[2] = ["SEMI_RACING_MUFFLER"];
+	MUFFLER_list[3] = ["RACING_MUFFLER"];
+	
+	static CLUTCH_list = Array(4);
+	CLUTCH_list[0] = ["NORMAL_CLUTCH"];
+	CLUTCH_list[1] = ["HIGH_CAPACITY_CLUTCH"];
+	CLUTCH_list[2] = ["TWIN_CLUTCH"];
+	CLUTCH_list[3] = ["TRIPLE_CLUTCH"];
+	
+	static FLYWHEEL_list = Array(4);
+	FLYWHEEL_list[1] = ["FLYWHEEL_LIGHT"];
+	FLYWHEEL_list[2] = ["FLYWHEEL_Cr_Mo"];
+	FLYWHEEL_list[3] = ["FLYWHEEL_LIGHT_Cr_Mo"];
+	
+	static PROPELLERSHAFT_list = Array(2);
+	PROPELLERSHAFT_list[1] = ["PROPELLERSHAFT"];
+	
+	static LSD_list = Array(6);
+	LSD_list[0] = ["NORMAL_LSD"];
+	LSD_list[1] = ["1WAY_LSD"];
+	LSD_list[2] = ["15WAY_LSD"];
+	LSD_list[3] = ["2WAY_LSD"];
+	LSD_list[4] = ["VARIABLE_LSD"];
+	LSD_list[5] = ["AYCC_LSD"];
+	
+	static FRONTTIRE_list = Array(13);
+	FRONTTIRE_list[0] = ["FRONTTIRE_NORMAL_ECONOMY"];
+	FRONTTIRE_list[1] = ["FRONTTIRE_NORMAL_COMFORT"];
+	FRONTTIRE_list[2] = ["FRONTTIRE_NORMAL_STREET"];
+	FRONTTIRE_list[3] = ["FRONTTIRE_SPORTS_HARD"];
+	FRONTTIRE_list[4] = ["FRONTTIRE_SPORTS_MEDIUM"];
+	FRONTTIRE_list[5] = ["FRONTTIRE_SPORTS_SOFT"];
+	FRONTTIRE_list[6] = ["FRONTTIRE_RACING_SUPER_HARD"];
+	FRONTTIRE_list[7] = ["FRONTTIRE_RACING_HARD"];
+	FRONTTIRE_list[8] = ["FRONTTIRE_RACING_MEDIUM"];
+	FRONTTIRE_list[9] = ["FRONTTIRE_RACING_SOFT"];
+	FRONTTIRE_list[10] = ["FRONTTIRE_RACING_QUALIFY"];
+	FRONTTIRE_list[11] = ["FRONTTIRE_DIRT"];
+	FRONTTIRE_list[12] = ["FRONTTIRE_SNOW"];
+	
+	static REARTIRE_list = Array(13);
+	REARTIRE_list[0] = ["REARTIRE_NORMAL_ECONOMY"];
+	REARTIRE_list[1] = ["REARTIRE_NORMAL_COMFORT"];
+	REARTIRE_list[2] = ["REARTIRE_NORMAL_STREET"];
+	REARTIRE_list[3] = ["REARTIRE_SPORTS_HARD"];
+	REARTIRE_list[4] = ["REARTIRE_SPORTS_MEDIUM"];
+	REARTIRE_list[5] = ["REARTIRE_SPORTS_SOFT"];
+	REARTIRE_list[6] = ["REARTIRE_RACING_SUPER_HARD"];
+	REARTIRE_list[7] = ["REARTIRE_RACING_HARD"];
+	REARTIRE_list[8] = ["REARTIRE_RACING_MEDIUM"];
+	REARTIRE_list[9] = ["REARTIRE_RACING_SOFT"];
+	REARTIRE_list[10] = ["REARTIRE_RACING_QUALIFY"];
+	REARTIRE_list[11] = ["REARTIRE_DIRT"];
+	REARTIRE_list[12] = ["REARTIRE_SNOW"];
+	
+	static NOS_list = Array(2);
+	NOS_list[1] = ["NOS"];
+	
+	static SUPERCHARGER_list = Array(2);
+	SUPERCHARGER_list[1] = ["SUPERCHARGER"];
+	
+	static WHEEL_list = Array(1);
+	
+	static WING_list = Array(1);
+	
+	static STIFFNESS_list = Array(2);
+	STIFFNESS_list[1] = ["STIFFNESS"];

```

Comment:
Is this correct? You're wrapping every element in an array when not expected, which causes an extra `ARRAY_CONST_OLD`